src/WebJobs.Script/Scale/HostPerformanceManager.cs

```diff
@@ -113,59 +103,17 @@ public async Task<IActionResult> TryHandleHealthPingAsync(HttpRequest request, I
+            if (throttleManager != null)
             {
-                string formattedLoadHistory = string.Join(",", hostProcessStats.CpuLoadHistory);
-                logger?.HostProcessCpuStats(_environment.GetEffectiveCoresCount(), formattedLoadHistory, Math.Round(hostProcessStats.CpuLoadHistory.Average()), Math.Round(hostProcessStats.CpuLoadHistory.Max()));
-            }
-
-            double workersAverageCpuTotal = 0;
-            var dispatcher = GetDispatcherAsync();
-            if (dispatcher != null)
-            {
-                // If OOP, check worker stats
-                var workerStatuses = await dispatcher.GetWorkerStatusesAsync();
-                if (workerStatuses.All(p => p.Value.ProcessStats.CpuLoadHistory.Count() > MinSampleCount))
-                {
-                    // first compute the average CPU load for each worker
-                    var averageWorkerCpuStats = new List<double>();
-                    foreach (var currentStatus in workerStatuses)
-                    {
-                        // take the last N samples
-                        var workerProcessStats = currentStatus.Value.ProcessStats;
-                        int currWorkerProcessCpuStatsCount = workerProcessStats.CpuLoadHistory.Count();
-                        var currWorkerCpuStats = workerProcessStats.CpuLoadHistory.Skip(currWorkerProcessCpuStatsCount - MinSampleCount).Take(MinSampleCount).Average();
-                        averageWorkerCpuStats.Add(currWorkerCpuStats);
-                    }
-
-                    // compute the final average CPU total across all workers
-                    workersAverageCpuTotal = averageWorkerCpuStats.Sum();
-                }
-            }
-
-            // calculate the aggregate load of host + workers (if OOP)
-            int hostProcessCpuStatsCount = hostProcessStats.CpuLoadHistory.Count();
-            if (hostProcessCpuStatsCount > MinSampleCount)
-            {
-                // compute the aggregate average CPU usage for host + workers for the last MinSampleCount samples
-                var hostAverageCpu = hostProcessStats.CpuLoadHistory.Skip(hostProcessCpuStatsCount - MinSampleCount).Take(MinSampleCount).Average();
-                var aggregateAverage = Math.Round(hostAverageCpu + workersAverageCpuTotal);
-                logger?.HostAggregateCpuLoad(aggregateAverage);
-
-                // if the average is above our threshold, return true (we're overloaded)
-                var adjustedThreshold = _healthMonitorOptions.Value.CounterThreshold * 100;
-                if (aggregateAverage >= adjustedThreshold)
-                {
-                    logger?.HostCpuThresholdExceeded(aggregateAverage, adjustedThreshold);
-                    if (exceededCounters != null)
-                    {
-                        exceededCounters.Add("CPU");
-                    }
-                    return true;
-                }
+                var status = throttleManager.GetStatus();
```

Comment:
All the previous process monitoring moved into the WebJobs SDK and is part of dynamic concurrency throttles. Thus the code here is much simpler now - it just queries the throttle manager for current throttle state, and all throttles are consulted (CPU/Memory/ThreadStarvation/WorkerChannel/etc.).

Reply:
For all the logging extension event definitions moved, their event names have remained the same, though event ID numbers may have changed, since in moving into the respective logger extensions in the SDK, the numbers were rebased to the sequence present in the SDK.